File: smash-ranks-android/app/src/main/java/com/garpr/android/misc/CrashlyticsWrapper.java
package com.garpr.android.misc;

import android.support.annotation.NonNull;

public interface CrashlyticsWrapper {

    void initialize(final boolean disabled);

    void log(final int priority, final String tag, final String msg);

    void logException(@NonNull final Throwable tr);

    void setBool(@NonNull final String key, final boolean value);

    void setInt(@NonNull final String key, final int value);

}

File: smash-ranks-android/app/src/main/java/com/garpr/android/wrappers/CrashlyticsWrapper.kt
package com.garpr.android.wrappers

interface CrashlyticsWrapper {

    fun initialize(disabled: Boolean)

    fun log(priority: Int, tag: String, msg: String)

    fun logException(tr: Throwable)

    fun setBool(key: String, value: Boolean)

    fun setInt(key: String, value: Int)

    fun setString(key: String, value: String)

}
